### PR TITLE
BlazorWebView

### LINKED ISSUE #12: BlazorWebView
Document the `BlazorWebView` control, introduced in P4, which is used to host a Blazor web app in a .NET MAUI app.

For more info, see https://devblogs.microsoft.com/aspnet/asp-net-core-updates-in-net-6-preview-4/#net-maui-blazor-apps

Usage:

Add a`BlazorWebView` to any page and point to the root of the Blazor app:

```
	<BlazorWebView 
	    HostPage="wwwroot/index.html"
	    Services="{StaticResource Services}">
	    <BlazorWebView.RootComponent>
	        <RootComponent 
	            Selector="#app"
	            ComponentType="{x:Type local:Main}"  />
	    </BlazorWebView.RootComponent>
	</BlazorWebView>
```

### PR DESCRIPTION
**Pull request by davidbritch**
Replaces #134 

Fixes #12

### CODE REVIEW

File: docs/user-interface/controls/blazorwebview.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+1. Modify your `MauiProgram` class to add `using` directives for the `Microsoft.AspNetCore.Components.WebView.Maui` and `Microsoft.Extensions.DependencyInjection` namespaces:
+
+    ```csharp
+    using Microsoft.AspNetCore.Components.WebView.Maui;
+    using Microsoft.Extensions.DependencyInjection;
+    ```
+
+    > [!NOTE]
+    > Blazor requires service scopes, which are only supported with the `Microsoft.Extensions.DependencyInjection` library.
```
Comment (Eilon) [inline docs/user-interface/controls/blazorwebview.md]: I think this note is no longer needed; the default MAUI stuff has everything you need.

File: docs/user-interface/controls/blazorwebview.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                             fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+                       });
+
+                 builder.Services.AddBlazorWebView();
+                 // Register any app services on the IServiceCollection object
+                 // e.g. builder.Services.AddSingleton<WeatherForecastService>();
+
+                 return builder.Build();
+           }
+    }
```
Comment (Eilon) [inline docs/user-interface/controls/blazorwebview.md]: No code change, just making indentation consistent:

```suggestion
    public static class MauiProgram
    {
        public static MauiApp CreateMauiApp()
        {
            var builder = MauiApp.CreateBuilder();
            builder
                .RegisterBlazorMauiWebView()
                .UseMauiApp<App>()
                .ConfigureFonts(fonts =>
                {
                    fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
                });

            builder.Services.AddBlazorWebView();
            // Register any app services on the IServiceCollection object
            // e.g. builder.Services.AddSingleton<WeatherForecastService>();

            return builder.Build();
        }
    }
```

**Review by Eilon — CHANGES_REQUESTED** [review]
Just 2 small comments and then I think it's good to go!

**Review by Eilon — APPROVED** [review]
Ship it!

**Comment by Eilon** [comment]
Apparently my review isn't good enough to approve ☹️ Who can give me more permission in this repo? I'll probably end up writing a doc or two as well.

**Comment by davidbritch** [comment]
@Eilon I've given you write access. That seems to have made your approval count!

**Comment by Eilon** [comment]
Perfect! Let's get this published! 😄

**Pull request merged**